.drone.yml

```diff
@@ -1,116 +1,122 @@
-pipeline:
-  build_are:
-    image: docker:18.03.1-ce
-    environment:
-      - DOCKER_HOST=tcp://172.17.0.1:2375
-    commands:
-      - docker build -t app-$${DRONE_COMMIT_SHA} .
-    when:
-      event: pull_request
+---
+kind: pipeline
+name: default
+type: kubernetes
 
-  build_are:
-    image: docker:18.03.1-ce
-    environment:
-      - DOCKER_HOST=tcp://172.17.0.1:2375
-    commands:
-      - docker build -t app-$${DRONE_COMMIT_SHA} .
-    when:
-      branch: master
-      event: push
+steps:
+- name: build_are
+  pull: if-not-exists
+  image: docker:dind
+  commands:
+  - n=0; while [ "$n" -lt 60 ] && [ ! -e /var/run/docker.sock ]; do n=$(( n + 1 )); sleep 1; done
+  - docker build -t app-$${DRONE_COMMIT_SHA} .
+  environment:
+    IMAGE_NAME: app-$${DRONE_COMMIT_SHA}
```

Comment:
This can be removed as its not used as far as I can see

Reply:
removed environment, and variable